packages/components/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,259 @@
+import { Colors } from '@department-of-veterans-affairs/mobile-tokens'
+// import { HapticFeedbackTypes } from 'react-native-haptic-feedback'
+import { Text, TextStyle, View, ViewStyle } from 'react-native'
+//      ^ ScrollView,
+import React, { FC } from 'react'
+//                ^ , RefObject, useEffect, useState
+
+// import { triggerHaptic } from 'utils/haptics'
```

Comment:
Added AC to expand/collapse ticket to clean this stuff up since [the Slack thread was a little slow in confirming we don't intend to have a follow-up ticket (pre-MVP) for adding support](https://adhoc.slack.com/archives/C05DQPHUVNZ/p1710874405242539?thread_ts=1709747499.865419&cid=C05DQPHUVNZ).